### DISCUSSION TITLE
Secure the free version of tinypilot

### DISCUSSION THREAD

**Discussion by Th3Gr1ff** [tiny-pilot/tinypilot#1163] [Ideas]
If security is a need then you can use openziti to create a closed network interface to the PI. so only someone with a valid certificate and the ziti software can access the TP web UI. which will be far more secure than any plain HTTPS since only designated people can access it. make the HTTP listen only on localhost not the bound IP. boom, more secure and FREE

**Comment by icsy7867** [comment]
I'm new and looking at the product so I'm not quite sure.  But doesn't this use nginx as a reverse proxy?  I think this is fairly trivial to setup if so.
